common/data_refinery_common/migrations/0012_experiment_platform_accession_codes.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 2.1.5 on 2019-01-30 21:39
+
+import django.contrib.postgres.fields
+from django.db import migrations, models
+
+def update_cached_values(apps, schema_editor):
+    """ """
+
+    Experiment = apps.get_model('data_refinery_common', 'Experiment')
+    for experiment in Experiment.objects.all():
+        # Model methods can't be used during migrations. :(
+        # https://stackoverflow.com/a/37685925/1135467
+        experiment.platform_accession_codes = list(set([sample.platform_accession_code for sample in experiment.samples.all()]))
```

Comment:
I think this should be:
```
experiment.platform_accession_codes = get_platform_accession_codes()
```

Reply:
You can't use model methods during migrations. There's already a comment and link to an explanation directly above this line.

Reply:
Hah good call, I didn't notice that.

Reply:
and even then it would be `experiment.get_platform_accession_codes()`

Reply:
:-P